charts/consul/templates/telemetry-collector-deployment.yaml

```
@@ -34,6 +34,8 @@ spec:
         # This annotation tells the endpoints controller that this pod was injected even though it wasn't. The
         # endpoints controller would then sync the endpoint into Consul
         "consul.hashicorp.com/connect-inject-status": "injected"
+        # Signals that we should use the default namespace, ignoring all other namespace selection logic.
```

Not sure I understood this comment in the context of the changes. Maybe the word `default` is tripping me up. I was thinking something more along these lines:
```suggestion
        # Signals to the endpoints controller that we should force Consul NS creation, since we bypass the mesh webhook.
```

---

Ah whoops. This was from an approach I tried and abandoned: forcing the CTC to be deployed to the default namespace